packages/zoe/src/contractFacet/internal-types.js

```diff
@@ -30,6 +21,6 @@
  * @property {(instanceAdminFromZoe: ERef<ZoeInstanceAdmin>,
  *     instanceRecordFromZoe: InstanceRecord,
  *     issuerStorageFromZoe: IssuerRecords,
- *     privateArgs: Object=,
+ *     privateArgs?: Object,
```

Comment:
I'm curious - what's the benefit of this change?

Reply:
Once I turn on `@ts-check`, the `Object=` form provokes a ts complain on hover in vscode:

![image](https://user-images.githubusercontent.com/273868/134053110-ea7d89d6-9f7a-44f7-8bfd-314a7119eece.png)

Reply:
Interesting! Thanks for the explanation!